src/components/Checkbox/Checkbox.tsx

```diff
@@ -14,6 +14,7 @@ import { generateUniqueId } from '../_private/UniqueId';
 import { Text } from '../Text';
 import { motion, useMotionValue, useTransform } from 'framer-motion';
 import 'focus-visible';
+import { screenreaderOnlyStyles } from '../../styles/screenreaderOnly';
```

Comment:
You'll see below where this is coming from, but is another case where it'd be nice to add things (even full classes) to the theme (https://github.com/lifeomic/chroma-react/issues/47)